### ISSUE TITLE
Closing Oracle connection if no activity 

### ISSUE THREAD

**Issue by tksd123** [erdemcer/kafka-connect-oracle#9]
Hi,

When there is no activity for few minutes, the connector is throwing an error continuously and nothing works anymore.

`ERROR Error at poll java.sql.SQLRecoverableException: No more data to read from socket (com.ecer.kafka.connect.oracle.OracleSourceTask)`

Any idea? I'm keen to help if I can :)

Thank you

Nick

**Comment by erdemcer** [comment]
Hi,
Which Oracle version and JDBC driver are you using ? 
Thanks

**Comment by tksd123** [comment]
Hi, 

Thanks for replying :-)

Oracle 12C and ojdbc8

**Comment by erdemcer** [comment]
Hi,
The problem seems related with network . Could you please check your network protocol properties in listener.ora and tnsnames.ora . And also check alert.log file is there any error related with this issue . I will test on my side .
Thanks

**Comment by tksd123** [comment]
Hi,

Thanks for your time.
What should I check excatly into those files? I have no custom configuration.

I'm running Oracle and Kafka connect in containers do you think it can be the root cause?
The oracle image is this one : https://github.com/MaksymBilenko/docker-oracle-12c

**Comment by erdemcer** [comment]
Yes it is possible, for some reason your connection is interrupted. Does connector capture any dml before error?

I have not tested connector on database on container. Do you have any chance to setup database on another platform like vm or physical?

**Comment by tksd123** [comment]
I've tried to connect it directly to oracle hosted on a vm but nothing better :(

Yes it captured some DML before dying

**Comment by 475350551** [comment]
i hava the same problem,  is there any solution ?thanks

**Comment by MZ-Hyojin** [comment]
same issue too

**Comment by Ppojin** [comment]
Same issue too
